frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -50,9 +50,9 @@ export const StyledSidebar = styled.section<StyledSidebarProps>(
     const maxWidth = isCollapsed ? 0 : Math.min(550, window.innerWidth * 0.9)
 
     return {
-      // Nudge the sidebar by 2px so the header decoration doesn't go below it
       position: "relative",
-      top: adjustTop ? "2px" : "0px",
+      // Nudge the sidebar by 2px so the header decoration doesn't go below it
+      top: adjustTop ? theme.sizes.headerDecorationHeight : "0px",
```

Comment:
nit: could "0px" be `theme.sizes.none` or is `0` and `0px` something different?

Reply:
yep, I think we can use that there as well